src/lib/Client.js

```diff
@@ -340,13 +340,18 @@ class KlasaClient extends Discord.Client {
 	}
 
 	/**
-	 * The owner for this bot
-	 * @since 0.1.1
-	 * @type {?KlasaUser}
+	 * The owners for this bot
+	 * @since 0.5.0
+	 * @type {Set<KlasaUser>}
 	 * @readonly
 	 */
-	get owner() {
```

Comment:
If you wanted to keep some backwards compatibility, you could do
```js
get owner() {
    return this.options.owners.length === 1 ? this.users.get(this.options.owners.values().next().value) : null;
}
```

Reply:
No, that'd make many things harder, overloaded outputs are a pain to maintain.